### PR TITLE
Move integration test to  azure spring data test

### PR DESCRIPTION
**Pull request by chenrujun**

### CODE REVIEW

File: sdk/cosmos/azure-spring-data-cosmos-core/src/samples/java/com/azure/cosmos/PageableRepositoryCodeSnippet.java [hunk 1]
```diff
@@ -1,39 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-spring-data-cosmos-core/src/samples/java/com/azure/cosmos/PageableRepositoryCodeSnippet.java:1]: Why are we removing this code snippet ?
Is it not getting used anywhere ?

Comment (chenrujun) [reply sdk/cosmos/azure-spring-data-cosmos-core/src/samples/java/com/azure/cosmos/PageableRepositoryCodeSnippet.java:1]: Yes, it not used. I deleted it and integration test passed.

Comment (chenrujun) [reply sdk/cosmos/azure-spring-data-cosmos-core/src/samples/java/com/azure/cosmos/PageableRepositoryCodeSnippet.java:1]: It used in readme, I'll recover it.

**Pull request merged**